CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- `requiredFields` property to the form set `*` in label of required fields
```

Comment:
I think it would be good to include something that indicates that it's only marking the label with an asteriks. Otherwise the consumer of this library might think it will also do validation.

Reply:
@neotrow Done. Better for you now? :D